ovs/proto_trace.go

```diff
@@ -113,14 +117,15 @@ type ProtoTrace struct {
 	InputFlow       *DataPathFlows
 	FinalFlow       *DataPathFlows
 	DataPathActions DataPathActions
+	FlowActions     []string
```

Comment:
Should this follow the pattern set by the DataPathActions type above where it has an interface that implements TextUnmarshaler and all that? The thing is, none of that was ever really used since [prototrace UnmarshalText just creates an instance of it directly][1] bypassing all of that.

I'm wondering how much design has really gone into this library and whether we need to do better.

[1] https://github.com/digitalocean/go-openvswitch/blob/9d885f66269d368bdac4b34440e6b7640531efbf/ovs/proto_trace.go#L126-L128

Reply:
I prefer to keep it simple, and if we eventually need it we can add it

Reply:
Simple was what I was after originally. However, I just hope we don't paint ourselves into a corner. Would we end up in a situation where we have to change the API that consumers of this library use?